index.js

```diff
@@ -109,15 +109,15 @@ function disableFilters(solutions) {
 
 async function main() {
     indexJson = await $.ajax({
-        url: "index.json"
+        url: "index.json" + (Math.random()*10000)
```

Comment:
You have to add this as a parameter e.g. "index.json?cb=" + (Math.random()*10000)

Otherwise, you would be downloading e.g. index.json1357 and this does not exist on the server. The correct URL would be "index.json?cb=1357"